refinery/core/utils.py

```diff
@@ -983,8 +983,7 @@ def move_obj_to_front(obj_arr, match_key, match_value):
 
 
 def api_error_response(error_message, http_status_code):
-    """Return and log error for Django Rest Framework API calls"""
-    logger.error(error_message)
+    """Return a standardized error for Django Rest Framework API calls"""
     return Response({'Error': error_message}, status=http_status_code)
```

Comment:
Would we like to keep the logging, in addition to returning the error response? Or maybe its handled elsewhere, or maybe it would be log clutter?